### ISSUE TITLE
[med] : Many pages: Chrome + Jaws : Screen reader is not announcing the search results information.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8616]
## Action Performed:
1. Using Chrome, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to any workspace using Tab key and press using Enter to select it.
4. Navigate to Manage members button using Tab key and press Enter on it.
5.Select members and navigate to invite button using Tab key and press Enter on it.
6. Invite New member window and type any input value in search editable text box.
7. Observe the screen reader announcement.

## Expected Result:
Screen reader should read the information about the search results. if no results found then screen reader should read "no results found".

## Actual Result:
SCreen reader is not read anything after searching the reults.

HTML-
`<input placeholder="Name, email, or phone number" autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" spellcheck="true" type="text" class="css-11aywtz r-13awgt0 r-tz77pi r-13qz1uu" value="hhj" style="border-radius: 8px; border-width: 0px; box-shadow: none; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; outline-width: 0px; padding: 0px 11px;" `selection="null">``

**Other Occurrence:**
Similar type of repro on PR #8019, #8375, #8535, #8751, #8818, #8818, #8494, #9144, #9351, #9472
>Similar type of repro on PR #7702 for currencies.

## Workaround:
Yes

## Area issue was found in:
Invite New members

## Failed WCAG checkpoints
1.3.1

## User impact:
The SR user cannot understand that how many search results are appearing on the screen.

## Suggested resolution:
Screen reader should announce the search results information.

Refer to -
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:**v1.1.54-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/163121286-ee91f157-9c3a-4aee-b453-951f7b74a655.mp4
